redhat_upgrade_tool/rollback/__init__.py

```diff
@@ -1,2 +1,10 @@
+
+import os
+
 rhel6_profile = "98c3edb"
 snapshot_metadata_file = "/boot/grub/snapshot.metadata"
+rollback_dir = "/boot/rollback"
+active_kernel_file = os.path.join(rollback_dir, '.active-kernel')
+all_kernels_file = os.path.join(rollback_dir, '.all-kernels')
+target_kernel_file = os.path.join(rollback_dir, '.target-kernel')
+snap_boot_files_file = os.path.join(rollback_dir, '.snap_boot_files')
```

Comment:
I would change the name of this variable, ```snap_boot_files_file```, I see why it's like this, although if someone is looking after a long time into that code, they might be confused.

But it is only a suggestion, up to you..

Reply:
I agree and I spent a time above this name already. But I still can't come up with any good name at all, soI will keep this for now.